pkg/controllers/node/hostendpoints.go

```diff
@@ -28,6 +28,44 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	defaultProfileName = "allow"
```

Comment:
Actually going to pull this name in from libcalico-go https://github.com/projectcalico/libcalico-go/pull/1207